_pages/techfaqs.md

```diff
@@ -334,3 +334,50 @@ unknown parties.
 Relative paths, however, are permitted in both GEDCOM and GEDZIP files and so may be more convenient to
 use in both cases.  Any tool that converts FamilySearch GEDCOM 7.0 to GEDZIP should convert local file URLs
 to relative paths within the resulting GEDZIP file.
+
+# How do I record the URL of a source?
+
+While `EVENT_DETAIL` and `REPOSITORY_RECORD` both allow a URL to be placed in a `WWW` substructure,
+`SOURCE_CITATION`, `SOURCE_RECORD`, and `SOURCE_REPOSITORY_CITATION` do not.
+
+Some applications might put the source URL for an `EVENT_DETAIL` in a `WWW` substructure
+in parallel to a `SOURCE_CITATION` as follows:
+
+```
+1 DEAT
+2 DATE 14 DEC 1799
+2 WWW https://www.findagrave.com/memorial/1075
+2 SOUR @FindAGraveSourceRecord@
+3 PAGE Memorial: 1075
+```
```

Comment:
I would prefer we remove this example as it is misleading, possibly even incorrect.

Many events in the last 20 years have URLs for the event itself, containing an invitation, registry, photo album, etc. Many within the last 10 years have a URL as their only location: the online meeting address. These are worth recording as they convey something about the event itself (how it was held) and can sometimes be used to look up sources in web archive repositories (like archive.org and [many others](https://en.wikipedia.org/wiki/List_of_Web_archiving_initiatives)). These are separate from the URL of sources: a wayback machine URL is not the URL of the event, it is the URL of the record in the Internet Archive, and both might be separate from the URL of a separately-created record in an online or in-person archive or database.